### ISSUE TITLE
Bugs cause Pokollection and Recommendations not to load due to pokemon name issues when fetching image URLs

### ISSUE THREAD

**Issue by baker-ling** [mdbow22/I-Choose-Who#32]

**Comment by baker-ling** [comment]
Response body:
{
  "code": "ERR_UNESCAPED_CHARACTERS"
}

**Comment by baker-ling** [comment]
Probably caused by "Farfetch'd"

**Comment by baker-ling** [comment]
Here are all the pokemon who seem like they might cause this bug in the future:
![image](https://user-images.githubusercontent.com/69489054/141732608-6fd24c66-a2e7-4c43-bb21-7e07ff6ef5be.png)

**Comment by mdbow22** [comment]
It looks like periods (.) just get removed, and spaces are converted to hyphens (-).

**Comment by baker-ling** [comment]
What happens to the apostrophe? Just removed like periods?

**Comment by mdbow22** [comment]
Yeah, the apostrophe is just removed so farfetch'd -> farfetchd, mr. mime -> mr-mime. Hyphens are unchanged so Ho-oh and Porgyon-Z should not be an issue.

**Comment by baker-ling** [comment]
TODO: Add error handling to getPics. Provide a placeholder image URL when API call fails.
